### DISCUSSION TITLE
colorize the right click specific menu

### DISCUSSION THREAD

**Discussion by arifdarmawan9** [valinet/ExplorerPatcher#3247] [Ideas]
I want to make the right click menu colorized ... for example when  I right click at a file under the home folder, I right click the file and searching the "open location folder" ... it takes time to search this menu...so I want to colorize the frequently used menu to find it faster...my eyes are bad...:)

**Comment by pyrates999** [comment]
This is beyond what EP is for, sorry.

Try this: https://support.microsoft.com/en-us/windows/change-color-contrast-in-windows-fedc744c-90ac-69df-aed5-c8a90125e696
